src/components/StaffList/StaffList.scss

```diff
@@ -0,0 +1,8 @@
+.react-autosuggest__input {
+  width: 100%;
+}
+
+$light-gray: #aaa;
```

Comment:
super nit: usual convention is sass variables go at the top of the file

Reply:
There should be a separate ```variables``` partial to make the variables reusable across the repo (see https://github.com/edx/prospectus/blob/master/src/sass/base/_variables.scss).

Reply:
👍